.github/tracing/tempo.yaml

```diff
@@ -0,0 +1,24 @@
+server:
+  http_listen_port: 3200
+
+distributor:
+  receivers:                           
+    otlp:
+      protocols:
+        http:
+        grpc:
+
+ingester:
+  max_block_duration: 5m               # cut the headblock when this much time passes. this is being set for demo purposes and should probably be left alone normally
```

Comment:
Is this yaml auto-formatted? Seems like a lot of space before the comment :shrug:

Reply:
Interestingly I grabbed this file from an example in the OTEL repo and then edited the values for our needs. Unsure why all that space is there